xline/src/storage/lease_store/mod.rs

```diff
@@ -370,42 +374,31 @@ where
     }
 
     /// Attach key to lease
+    #[cfg(test)]
```

Comment:
Why should we add `#[cfg(test)]` here ?

Reply:
Because after this refactor, other store's can directly call `attach` by `lease_collection, and `lease_store` itself didn't need `attach`

Reply:
You can move this `attach` method in the `test` mod, like:
```Rust
#[cfg(test)]
mod test {
    ....
    fn attach<DB: StorageApi>(lease_store: &LeaseStore<DB>, lease_id: i64, key: Vec<u8>) -> Result<(), ExecuteError> {
        lease_store.inner.lease_collection.attach(lease_id, key)
    }

    #[tokio::test(flavor = "multi_thread", worker_threads = 10)]
    async fn test_lease_storage() -> Result<(), Box<dyn Error>> {
        let db = DBProxy::open(&StorageConfig::Memory)?;
        let lease_store = init_store(db);
        ....
        let attach_non_existing_lease = attach(&lease_store, 0, "key".into());
        assert!(attach_non_existing_lease.is_err());
        let attach_existing_lease = attach(&lease_store, 1, "key".into());
        assert!(attach_existing_lease.is_ok());
        ....
    }
....
}
```

Reply:
Fixed